named_args.go

```diff
@@ -107,6 +107,19 @@ func rawState(l *sqlLexer) stateFn {
 			return singleQuoteState
 		case '"':
 			return doubleQuoteState
+		case ':':
+			nextRune, _ := utf8.DecodeRuneInString(l.src[l.pos:])
+			prevRune := rune(0)
+			if l.pos > 1 {
+				prevRune, _ = utf8.DecodeRuneInString(l.src[l.pos-2:])
+			}
+			if nextRune != ':' && prevRune != ':' && (isLetter(nextRune) || nextRune == '_') {
```

Comment:
It seems to me that we can omit the first check nextRune != ':' as there will be a more specific check next, but in addition I would like to say that this check is much easier than the next ones, and will cut off cast types a little faster.... But type casts are not done so often to leave this prevenient check.

What do you think, should I remove `nextRune != ':' &&` ?